test/tests/misc/solution_invalid/tests

```diff
@@ -133,4 +133,21 @@
       detail = 'restarting from the output data'
     []
   []
+
+  [show_solution_warning_history]
+    type = 'RunApp'
+    input = 'solution_invalid_timehistory.i'
+    design = 'SolutionInvalidity.md SolutionInvalidityOutput.md'
+    expect_out = "Solution Invalid Warnings History:
+-----------------------------------------------------------------------------------------
+|                   Object                    | Time | Timeinterval Count | Total Count |
```

Comment:
```suggestion
|                   Object                    | Time | Timestep Count | Total Count |
```
?

Reply:
a time interval would be measure in simulation time (seconds most likely). 

maybe you prefer `Step interval` ?